server/command.go

```diff
@@ -85,11 +85,61 @@ func getCommand() *model.Command {
 		DisplayName:      "Bitbucket",
 		Description:      "Integration with Bitbucket.",
 		AutoComplete:     true,
+		AutocompleteData: getAutocompleteData(),
 		AutoCompleteDesc: "Available commands: connect, disconnect, todo, me, settings, subscribe, unsubscribe, help",
 		AutoCompleteHint: "[command]",
 	}
 }
 
+func getAutocompleteData() *model.AutocompleteData {
+	bitbucket := model.NewAutocompleteData("bitbucket", "[command]", "Available commands: connect, disconnect, todo, me, settings, subscribe, unsubscribe, help")
+
+	connect := model.NewAutocompleteData("connect", "", "Connect your Mattermost account to your Bitbucket account")
+
+	bitbucket.AddCommand(connect)
+
+	disconnect := model.NewAutocompleteData("disconnect", "", "Disconnect your Mattermost account from your Bitbucket account")
+	bitbucket.AddCommand(disconnect)
+
+	help := model.NewAutocompleteData("help", "", "Display Slash Command help text")
+	bitbucket.AddCommand(help)
+
+	todo := model.NewAutocompleteData("todo", "", "Get a list of unread messages and pull requests awaiting your review")
+	bitbucket.AddCommand(todo)
+
+	me := model.NewAutocompleteData("me", "", "Display the connected Bitbucket account")
+	bitbucket.AddCommand(me)
+
+	settings := model.NewAutocompleteData("settings", "[setting] [value]", "Update your user settings")
+
+	settingNotifications := model.NewAutocompleteData("notifications", "", "Turn notifications on/off")
+	settingValue := []model.AutocompleteListItem{{
+		HelpText: "Turn notifications on",
+		Item:     "on",
+	}, {
+		HelpText: "Turn notifications off",
+		Item:     "off",
+	}}
+	settingNotifications.AddStaticListArgument("", true, settingValue)
+	settings.AddCommand(settingNotifications)
+	bitbucket.AddCommand(settings)
+
+	subscribe := model.NewAutocompleteData("subscriptions", "[command]", "Available commands: list, add")
+	subscribeList := model.NewAutocompleteData("list", "", "List Subscription")
+	subscribe.AddCommand(subscribeList)
+	bitbucket.AddCommand(subscribe)
+
+	subscribeAdd := model.NewAutocompleteData("add", "owner[/repo] features", "subscribe to org/[repo]")
+	subscribeAdd.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
+	subscribeAdd.AddTextArgument("Comma-delimited list of one or more of: issues, pulls, pushes, creates, deletes, issue_comments, pull_reviews. Defaults to pulls,issues,creates,deletes", "[features] (optional)", `/[^,-\s]+(,[^,-\s]+)*/`)
+	subscribe.AddCommand(subscribeAdd)
+
+	unsubscribe := model.NewAutocompleteData("unsubscribe", "[unsubscribe] [owner/repo]", "Remove subscription for org/[repo]")
+	bitbucket.AddCommand(unsubscribe)
```

Comment:
`unsubscribe` should also be a subcommand of `subscriptions`. I would suggest using the same handler as in https://github.com/mattermost/mattermost-plugin-github/blob/4db6b6a461d7d1ec5e137ee8a744b6ced3ed4e2d/server/plugin/command.go#L224-L242